nilearn/datasets/func.py

```diff
@@ -1462,61 +1801,85 @@ def _fetch_development_fmri_functional(participants, data_dir, url, resume,
-                            skip_header=True, dtype=dtype, names=names)
+    osf_data = csv_to_array(
+        os.path.join(package_directory, "data", "development_fmri.csv"),
+        skip_header=True,
+        dtype=dtype,
+        names=names,
+    )
 
     funcs = []
     regressors = []
 
-    for participant_id in participants['participant_id']:
-        this_osf_id = osf_data[osf_data['participant_id'] == participant_id]
+    for participant_id in participants["participant_id"]:
+        this_osf_id = osf_data[osf_data["participant_id"] == participant_id]
         # Download regressors
-        confound_url = url.format(this_osf_id['key_r'][0])
-        regressor_file = [(confounds.format(participant_id),
-                           confound_url,
-                           {'move': confounds.format(participant_id)})]
-        path_to_regressor = _fetch_files(data_dir, regressor_file,
-                                         verbose=verbose)[0]
+        confound_url = url.format(this_osf_id["key_r"][0])
+        regressor_file = [
+            (
+                confounds.format(participant_id),
+                confound_url,
+                {"move": confounds.format(participant_id)},
+            )
+        ]
+        path_to_regressor = _fetch_files(
+            data_dir, regressor_file, verbose=verbose
+        )[0]
         regressors.append(path_to_regressor)
         # Download bold images
-        func_url = url.format(this_osf_id['key_b'][0])
-        func_file = [(func.format(participant_id, participant_id), func_url,
-                      {'move': func.format(participant_id)})]
-        path_to_func = _fetch_files(data_dir, func_file, resume=resume,
-                                    verbose=verbose)[0]
+        func_url = url.format(this_osf_id["key_b"][0])
+        func_file = [
+            (
+                func.format(participant_id, participant_id),
+                func_url,
+                {"move": func.format(participant_id)},
+            )
+        ]
+        path_to_func = _fetch_files(
+            data_dir, func_file, resume=resume, verbose=verbose
+        )[0]
         funcs.append(path_to_func)
     return funcs, regressors
 
 
 @fill_doc
-def fetch_development_fmri(n_subjects=None, reduce_confounds=True,
-                           data_dir=None, resume=True, verbose=1,
-                           age_group='both'):
-    """Fetch movie watching based brain development dataset (fMRI)
+def fetch_development_fmri(
+    n_subjects=None,
+    reduce_confounds=True,
+    data_dir=None,
+    resume=True,
+    verbose=1,
+    age_group="both",
+):
+    """Fetch movie watching based brain development dataset (fMRI).
 
-    The data is downsampled to 4mm resolution for convenience with a repetition time (TR)
-    of 2 secs. The origin of the data is coming from OpenNeuro. See Notes below.
+    The data is downsampled to 4mm resolution for convenience
+    with a repetition time (TR) of 2 secs.
```

Comment:
Again, unrelated to formatting but I realize some terms in the docstrings of public functions aren't linked to their glossary definitions (for example here TR and fMRI which could be replaced with :term:`fMRI` and :term:`TR`).
I used to add these when randomly encountering them even if not the main topic of the PR.
I think it could be nice to continue adding these references, either sporadicly as I used to do or in a more organized way. WDYT ?

Reply:
just to be sure the list of possible terms we can link to are in the glossary?

https://github.com/nilearn/nilearn/blob/main/doc/glossary.rst

Reply:
Yes exactly. 
If you encounter terms you think deserve one, you can always add more definitions of course !